mail_service/worker_notify_user.py

```diff
@@ -100,8 +100,8 @@ def __init__(self, job_info):
             logger.debug("Processing mail for failed build.")
             self.image_under_test = job_info.get("project_name")
 
-        self.project = self.job_info["namespace"] + "/" + \
-            self.job_info["job_name"]
+        self.project = self.job_info["namespace"].split("-")[0] + \
+            "/" + self.job_info["job_name"]
 
```

Comment:
@bamachrn:  IMO, keeping the name as it will be for actual built image make more sense. Say user has added r.c.o in docker config, then user just need to look at the repository name and `docker pull`. 

@rtnpro @dharmit @mohammedzee1000 : WDYT ?